src/days-2-4-adv/adv.py

```diff
@@ -49,3 +49,63 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+print (f"""
+    {me.currentRoom.name}:
+
+    {me.currentRoom.description}
+    """)
+while True:
+    cmd = input("which way?-> ")
+    if cmd == "q":
+        print ("""
+    ~~~~ until tomorrow ~~~~
+        """)
+        break
+    if cmd == "n":
+        if hasattr(me.currentRoom.n_to, "name"):
+            me.currentRoom = me.currentRoom.n_to
+            print (f"""
+    {me.currentRoom.name}:
+
+    {me.currentRoom.description}
+    """)
+        else:
+            print ("""
+    Yo you cant go that way dog, duh, ya dingus
+            """)
+    elif cmd == "e":
+        if hasattr(me.currentRoom.e_to, "name"):
+            me.currentRoom = me.currentRoom.e_to
+            print (f"""
+    {me.currentRoom.name}:
+
+    {me.currentRoom.description}
+    """)
+        else:
+            print ("""
+    Yo you cant go that way dog, duh, ya dingus
+            """)
+    elif cmd == "s":
+        if hasattr(me.currentRoom.s_to, "name"):
+            me.currentRoom = me.currentRoom.s_to
+            print (f"""
+    {me.currentRoom.name}:
+
+    {me.currentRoom.description}
+    """)
+        else:
+            print ("""
+    Yo you cant go that way dog, duh, ya dingus
+            """)
+    elif cmd == "w":
+        if hasattr(me.currentRoom.w_to, "name"):
+            me.currentRoom = me.currentRoom.w_to
+            print (f"""
+    {me.currentRoom.name}:
+
+    {me.currentRoom.description}
+    """)
+        else:
+            print ("""
+    Yo you cant go that way dog, duh, ya dingus
+            """)
```

Comment:
One thing I forgot to mention: What will your program do if it receives an input you haven't coded for yet? What do you want it to do?